source/speechXml.py

```diff
@@ -73,7 +73,7 @@ def toXmlLang(nvdaLang):
 StandAloneTagCommand = namedtuple("StandAloneTagCommand", ("tag", "attrs", "content"))
 
 def _escapeXml(text):
-	text = unicode(text).translate(XML_ESCAPES)
+	text = str(text).translate(XML_ESCAPES)

```

Comment:
Do we ever expect text to be something else than str? if not, we can safely omit wrapping this into str

```suggestion
	text = text.translate(XML_ESCAPES)
```